angular2/src/angular2TextMask.ts

```diff
@@ -74,5 +74,5 @@ export class MaskedInputDirective implements OnInit, ControlValueAccessor{
   exports: [MaskedInputDirective],
   imports: [CommonModule]
 })
-export class TextMaskModule {
-}
+export class TextMaskModule {}
+export {conformToMask} from 'text-mask-core/dist/textMaskCore'
```

Comment:
@PerfectPixel Is that really necessary? It feels to me that simply exposing the function is more flexible and convenient?